project/ParadoxSupport.scala

```diff
@@ -0,0 +1,98 @@
+      Def.task { Seq(
+        { context: Writer.Context ⇒ new UnidocDirective(allClasses, context.location.tree.label, context.properties) }
+      )}
+    }.value
+  )
+
+  class UnidocDirective(allClasses: IndexedSeq[String], page: Page, variables: Map[String, String]) extends InlineDirective("unidoc") {
+    def render(node: DirectiveNode, visitor: Visitor, printer: Printer): Unit = {
+      if (node.label.contains('.')) {
+        val fqcn = node.label
+        val label = fqcn.split('.').last
+        val source = new DirectiveNode.Source.Direct(fqcn)
+        val attributes = new org.pegdown.ast.DirectiveAttributes.AttributeMap()
+        val updatedNode = new DirectiveNode(DirectiveNode.Format.Inline, node.name, label, source, attributes, null,  new TextNode(label))
+        renderByFqcn(updatedNode, visitor, printer)
+      }
+      else {
+        renderByClassName(node.label, node, visitor, printer)
+      }
+    }
+
+    def renderByClassName(label: String, node: DirectiveNode, visitor: Visitor, printer: Printer): Unit = {
+      def syntheticNode(group: String, label: String, c: String): DirectiveNode = {
+        val syntheticSource = new DirectiveNode.Source.Direct(c)
+        val attributes = new org.pegdown.ast.DirectiveAttributes.AttributeMap()
+        new DirectiveNode(DirectiveNode.Format.Inline, group, null, null, attributes, null,
+          new DirectiveNode(DirectiveNode.Format.Inline, group + "doc", label, syntheticSource, node.attributes, c,
+            new TextNode(label)
+          ))
+      }
+
+      val scalaLabel = label.replaceAll("\\\\_", "_")
+      val labelWithoutGenericParameters = scalaLabel.split("\\[")(0)
+      val javaLabel = scalaLabel.replaceAll("\\[", "&lt;").replaceAll("\\]", "&gt;").replace('_', '?')
+      val matches = allClasses.filter(_.endsWith('.' + labelWithoutGenericParameters))
+      matches.size match {
+        case 0 =>
+          throw new java.lang.IllegalStateException(s"No matches found for $label.")
+        case 1 if matches(0).contains("adsl") =>
+          throw new java.lang.IllegalStateException(s"Match for $label only found in one language: ${matches(0)}")
+        case 1 =>
+          syntheticNode("scala", scalaLabel, matches(0)).accept(visitor)
+          syntheticNode("java", javaLabel, matches(0)).accept(visitor)
+        case 2 if matches.forall(_.contains("adsl")) =>
+          matches.foreach(m => {
+            if (!m.contains("javadsl"))
+              syntheticNode("scala", scalaLabel, m).accept(visitor)
+            if (!m.contains("scaladsl"))
+              syntheticNode("java", javaLabel, m).accept(visitor)
+          })
+        case n =>
+          throw new java.lang.IllegalStateException(
+            s"$n matches found for $label, but not javadsl/scaladsl: ${matches.mkString(", ")}. " +
+              s"You may want to use the fully qualified class name as @unidoc[fqcn] instead of @unidoc[${label}]."
```

Comment:
Error message is like:

> [error] Caused by: java.lang.IllegalStateException: 2 matches found for ClusterReceptionist, but not javadsl/scaladsl: akka.cluster.client.ClusterReceptionist, akka.cluster.typed.internal.receptionist.ClusterReceptionist. You may want to use the fully qualified class name as @unidoc[fqcn] instead of @unidoc[ClusterReceptionist].
> [error]         at akka.ParadoxSupport$UnidocDirective.renderByClassName(ParadoxSupport.scala:78)